everyvoice/wizard/basic.py

```diff
@@ -141,6 +141,27 @@ def sanitize_input(self, response):
         response = super().sanitize_input(response)
         return response.strip()
 
+    def can_mkdir(self, path: Path) -> bool:
+        """Make sure it's possible to create path, without leaving it behind."""
+        dirs_to_make = []
+        d = path
+        while not d.exists() and d not in (Path("/"), Path(""), Path(".")):
+            dirs_to_make.append(d)
+            d = d.parent
```

Comment:
Will this survive `path=="/somewhere_exist/doesntexist/doesnexists/../../a/b/c`?
I guess it depends on `d.parent`.

Reply:
Yes, this is exactly the use case I was working to support, and the reason I did this patch. Previously, validation would have left `/somewhere_exist/doesntexist/doesnexists/../../a/b` on the filesystem, now this while loop will add all the components to `dirs_to_make` and stop at `/somewhere_exist`, since it exists.

Reply:
And the finally block at the end of this function will remove them all so we leave no traces behind just for trying to validate that we can create this deep directory.